lib/src/components/NxThreatIndicator/NxThreatIndicator.tsx

```diff
@@ -21,9 +21,15 @@ const NxThreatIndicator: FunctionComponent<Props> =
           policyThreatLevel != null ? categoryByPolicyThreatLevel[policyThreatLevel] :
           'unspecified',
 
-        className = `nx-threat-indicator nx-threat-indicator--${category}`;
+        className = `nx-threat-indicator nx-threat-indicator--${category}`,
+        label = `threat level ${category}`;
 
-    return <NxFontAwesomeIcon className={className} icon={faCircle} />;
+    // @ts-ignore
```

Comment:
What's this?

Reply:
Is to suppress a warning that my IDE was throwing when adding role="presentation".

```
TS2322: Type '{ aria-hidden: string; aria-label: string; role: string; className: string; icon: IconDefinition; }' is not assignable to type 'IntrinsicAttributes & Props & { children?: ReactNode; }'.
  Property 'role' does not exist on type 'IntrinsicAttributes & Props & { children?: ReactNode; }'.
```

Reply:
> Is to suppress a warning that my IDE was throwing when adding role="presentation".

Ah, so it seems that's not really supported.  There's no guarantee that passing the `role` prop to the underlying third-party `FontAwesomeIcon` component will cause that component to set it on the `<svg>`. It works now but might not on a future upgrade.  Further, in testing again locally, it's no longer reading the label at all for me.  Maybe leaving the role alone (as img) was better.

Reply:
be5eb73cabe4fbff96707c1d047f2fd3fb4498a6
It was still being read correctly for me with VO, but yes, seems like a good idea to leave image to avoid future compatibility problems.